airgun/browser.py

```diff
@@ -670,7 +670,14 @@ def move_to_element(self, locator, *args, **kwargs):
         el = self.element(locator, *args, **kwargs)
         self.execute_script(
             "arguments[0].scrollIntoView(false);", el, silent=True)
-        ActionChains(self.selenium).move_to_element(el).perform()
+        try:
+            ActionChains(self.selenium).move_to_element(el).perform()
+        except selenium.common.exceptions.JavascriptException as e:
+            if "javascript error: Cannot read property 'left' of undefined" in e.msg:
+                self.logger.debug("Ignoring 'Cannot read property left of undefined' exception")
+                pass
```

Comment:
`pass` is "no operation". since you added `logger.debug` call, you don't need it :)

Reply:
ah